pkg/util/cluster/cluster.go

```diff
@@ -31,7 +35,7 @@ func DiscoverSriov(clients *testclient.ClientSet, operatorNamespace string) (*En
 
 		node := state.Name
 		for _, itf := range state.Status.Interfaces {
-			if itf.TotalVfs > 0 {
+			if itf.TotalVfs > 0 && isDriverSupported(itf.Driver) {
```

Comment:
For some Mellanox NICs, the initial value of TotalVfs is 0.

Reply:
So, having the driver is enough to understand if the device is a sriov one?
Because with non sriov devices we'll still have TotalVfs=0 there

Reply:
I think using the driver is enough for now, as we only support a few nic models.